pyproject.toml

```diff
@@ -18,8 +18,9 @@ classifiers = [
 python = ">=3.6.1"
 tqdm = "^4.36"
 numpy = "^1.17"
-torch = "~1.2"
+torch = "^1.2"
```

Comment:
pytorch does not follow semver and new minor versions include include many breaking changes, so this can easily break seqvec. Also at the time when i pinned pytorch, allennlp didn't properly specify the torch version, though this seems to have been fixed recently (allenai/allennlp#3886).

Reply:
Makes sense. 

However, SeqVec only uses 1 explicit torch function which is really unlikely to change. 
Given that and that allennlp seems to have fixed the issue on their side, I would strongly suggest that we don't limit ourselves now on the possibility that a future update might break something but rather fix the issue if it arises.